src/test/java/codeu/controller/LoginServletTest.java

```diff
@@ -80,8 +81,10 @@ public void testDoPost_ExistingUser() throws IOException, ServletException {
     Mockito.when(mockUserStore.isUserRegistered("test username")).thenReturn(true);
     User mockUser = Mockito.mock(User.class);
 
+    //set mock user username/password, pw is encrypted as would be in stored user
```

Comment:
Isn't this covered in your code below?

Reply:
Yeah, I guess that just didn't seem clear to me at first because I didn't understand how Mockito works, but I'll remove the comment since it doesn't seem necessary